libs/utils/src/cast_vote_records.ts

```diff
@@ -32,6 +38,57 @@ export function getCurrentSnapshot(cvr: CVR.CVR): Optional<CVR.CVRSnapshot> {
   );
 }
 
+/**
+ * Because there is no place for "Absentee" vs. "Precinct" ballot on the CDF
+ * for cast vote records, we shove it into the `OtherStatus` field of the
+ * `CVRSnapshot`. This tool formats that metadata, which can be spread into
+ * a `CVRSnapshot`.
+ */
+export function buildCVRSnapshotBallotStyleMetadata(
```

Comment:
Should this be called `buildCVRSnapshotBallotTypeMetadata` rather than `buildCVRSnapshotBallotStyleMetadata`, since ballot style has another meaning?

Reply:
Yes!